build/charts/flow-aggregator/values.yaml

```diff
@@ -133,7 +133,7 @@ clickHouse:
   commitInterval: "8s"
   # -- Credentials to connect to ClickHouse. They will be stored in a Secret.
   connectionSecret:
-    username : "clickhouse_operator"
```

Comment:
I assume that's just a cosmetic change?

Reply:
yes, I will move typos and this to another PR.